imports/api/inventories.js

```diff
@@ -3,13 +3,29 @@ import { Mongo } from 'meteor/mongo';
 import { check } from 'meteor/check';
 import { authCheck } from '../../utils/authorization';
 
-export const Inventories = new Mongo.Collection('inventories');
+const Inventories = new Mongo.Collection('inventories');
 
 // If use arrow function this.userId will return undefined
 Meteor.publish('inventories', function() {
 	return Inventories.find({ owner: this.userId }); //, { sort: { createdAt: -1 }});
 });
 
+const insertInventory = (name, callback) => {
+	if (callback) {
+		Inventories.insert({
+			name: name.trim(),
+			owner: this.userId,
```

Comment:
@indirap I looked into your question and it looks like the `this.userId` here is returning `undefined` so the item was inserted without the owner and so it doesn't show up in the list

Reply:
Aaah yes that makes a lot of sense. I'll make the `userId` a parameter. Thanks!